### ISSUE TITLE
django 4 incompatibility

### ISSUE THREAD

**Issue by WolfgangFahl** [ad-freiburg/qlever-ui#42]
To work around #38 i am trying to do the installation steps as per the documentation
```bash
python3 manage.py makemigrations —-merge && python manage.py migrate
```

which leads to 
ImportError: cannot import name 'url' from 'django.conf.urls' (/home/wf/.local/lib/python3.8/site-packages/django/conf/urls/__init__.py)

see https://stackoverflow.com/questions/70319606/importerror-cannot-import-name-url-from-django-conf-urls-after-upgrading-to

requirements.txt is obviously not specific enough to make sure the documentation instructions still work: 
```
django
requests
django-import-export
```

**Comment by WolfgangFahl** [comment]
django~=3.2.12
seems to work around this issue

**Comment by hannahbast** [comment]
Was already fixed by ea63c57912764ce992d0a2a803be5512df225388 long ago and #79 now makes sure that this cannot happen again.
